packages/orbit-components/src/InputGroup/InputGroup.stories.tsx

```diff
@@ -155,53 +155,41 @@ const selectOptionsError = [
   { value: "November", label: "November" },
   { value: "December", label: "December" },
 ];
-export const Error = ({ label, flex, error, help, selectValue }) => {
-  return (
-    <InputGroup
-      label={label}
-      flex={flex}
-      onChange={action("onChange")}
-      onFocus={action("onFocus")}
-      onBlur={action("onBlur")}
-    >
+
+export const Error: Story = {
+  render: ({ error, help, selectValue, ...args }) => (
+    <InputGroup {...args}>
       <InputField placeholder="DD" error={error} help={help} />
       <Select
         options={selectOptionsError}
         value={selectValue}
         placeholder="Month"
-        error="Something went wrong on month field"
+        error={error}
```

Comment:
Is this change expected?

Same for the one below on InputField (year)

Reply:
Yes, it was expected, I made the change intentionally. Before the error message was hardcoded so changing the value in controls wouldn't have any effect.

But thinking about it now, as I disabled type controls in the InputField stories so selected type couldn't be changed, I will remove the "error" control from this Error story so it cannot be changed. And with that, it can be hardcoded like it was previously.